cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -979,6 +979,19 @@ storage:
         [Time]
         NTP=169.254.169.123
 
+  - filesystem: root
+    path: /etc/ssh/sshd_config
+    mode: 0400
+    contents:
+      inline: |
+        Subsystem sftp internal-sftp
+        ClientAliveInterval 600
+        AuthenticationMethods publickey
+        UseDNS no
+        UsePAM yes
```

Comment:
This should be `no`?

Reply:
I tested that the proposed solution from https://github.com/coreos/bugs/issues/2114 works correctly even with PAM enabled.

Reply:
Ok, I didn't understand what was changed from the default config. But I see it's the line: `AuthenticationMethods publickey` that is added. Was `ClientAliveInterval` also changed? I see it has the value `180` on the node I have checked:

```
$ sudo cat /etc/ssh/sshd_config 
# Use most defaults for sshd configuration.
Subsystem sftp internal-sftp
ClientAliveInterval 180
UseDNS no
UsePAM yes
PrintLastLog no # handled by PAM
PrintMotd no # handled by PAM
```

Reply:
yes, I thought it makes sense, but let's revert this value, because change less the better.